<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BrickEmuPy</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="deviceWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>4</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
      <widget class="QWidget" name="sideWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>2</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="sideGridLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionStep"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenBrick"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSpeed">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Speed</string>
    </property>
    <addaction name="actionSpeedx01"/>
    <addaction name="actionSpeedx02"/>
    <addaction name="actionSpeedx05"/>
    <addaction name="actionSpeedx1"/>
    <addaction name="actionSpeedx2"/>
    <addaction name="actionSpeedx5"/>
    <addaction name="actionSpeedx10"/>
    <addaction name="actionSpeedMax"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
   <addaction name="menuSpeed"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="toolTip">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionOpenBrick">
   <property name="text">
    <string>Open Brick Game</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionSpacetronic">
   <property name="text">
    <string>Spacetronic (W030-001A)</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <actiongroup name="speedGroup">
    <action name="actionSpeedx01">
    <property name="factor" stdset="0">
      <double>10</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>x0.1</string>
    </property>
    </action>
    <action name="actionSpeedx02">
    <property name="factor" stdset="0">
      <double>5</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>x0.2</string>
    </property>
    </action>
    <action name="actionSpeedx05">
    <property name="factor" stdset="0">
      <double>2</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>x0.5</string>
    </property>
    </action>
    <action name="actionSpeedx1">
    <property name="factor" stdset="0">
      <double>1</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="checked">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>x1</string>
    </property>
    </action>
    <action name="actionSpeedx2">
    <property name="factor" stdset="0">
      <double>0.5</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>x2</string>
    </property>
    </action>
    <action name="actionSpeedx5">
    <property name="factor" stdset="0">
      <double>0.2</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>x5</string>
    </property>
    </action>
    <action name="actionSpeedx10">
    <property name="factor" stdset="0">
      <double>0.1</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>x10</string>
    </property>
    </action>
    <action name="actionSpeedMax">
    <property name="factor" stdset="0">
      <double>0</double>
    </property>
    <property name="checkable">
      <bool>true</bool>
    </property>
    <property name="text">
      <string>Maximum</string>
    </property>
    </action>
  </actiongroup>
  <action name="actionactionKeyCodes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>actionKeyCodes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>actionDebug</receiver>
   <slot>toggle()</slot>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>actionDebug</receiver>
   <slot>toggle()</slot>
  </connection>
  <connection>
   <sender>actionRun</sender>
   <signal>triggered()</signal>
   <receiver>actionDebug</receiver>
   <slot>toggle()</slot>
  </connection>
  <connection>
   <sender>actionDebug</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionRun</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
  <connection>
   <sender>actionDebug</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionStop</receiver>
   <slot>setDisabled(bool)</slot>
  </connection>
  <connection>
   <sender>actionDebug</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionPause</receiver>
   <slot>setDisabled(bool)</slot>
  </connection>
  <connection>
   <sender>actionDebug</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionStep</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
 </connections>
</ui>
